tools/cli/app.go

```diff
@@ -202,11 +215,40 @@ func NewCliApp() *cli.App {
 			Usage:       "Operate Temporal cluster",
 			Subcommands: newClusterCommands(),
 		},
+		{
+			Name:        "dataconverter",
+			Aliases:     []string{"dc"},
+			Usage:       "Data Converter",
+			Subcommands: newDataConverterCommands(),
+		},
 	}
+	app.Before = LoadPlugins
+	app.After = KillPlugins
 
 	// set builder if not customized
 	if cFactory == nil {
 		SetFactory(NewClientFactory())
 	}
 	return app
 }
+
+func LoadPlugins(ctx *cli.Context) error {
+	dc_plugin := ctx.String(FlagDataConverterPlugin)
+	if dc_plugin != "" {
+		dataConverter, err := NewDataConverterPlugin(dc_plugin)
+		if err != nil {
+			fmt.Printf("data converter: %v", err)
+			return err
+		}
```

Comment:
Did you mean?
```suggestion
			return fmt.Errorf("data converter: %w", err)
```

Reply:
No it's as intended. Surprisingly to me, `urfave/cli` does not print the error if one is returned here, so users would get a silent failure. Maybe this should be `fmt.Fatalf`?